lib/client.js

```diff
@@ -13,7 +13,7 @@ var Client = function (config) {
 	}
 
 	var handleResponse = function (response) {
-		if (response.statusCode !== 200 && response.statusCode !== 201) {
+		if (response.statusCode !== 200 && response.statusCode !== 201 && response.statusCode !== 202) {
```

Comment:
add test for this. Make sure that we don't `error` on 200, 201, 202 and that we do `error` on other things...